packages/react/mixin.server.js

```diff
@@ -56,28 +53,25 @@ class ReactPlugin extends Mixin {
       .then((element) =>
         this.fetchData({}, element).then((data) => ({ element, data }))
       )
-      .then((result) => {
-        const { element, data } = result;
+      .then(({ element, data }) => {
         const markup = renderToString(element);
         const helmet = Helmet.renderStatic();
-        this.enhanceData(Object.assign({}, data, { markup, helmet }))
-          .then(template)
-          .then((document) => {
-            const routerContext = this.routerContext;
-            if (routerContext.miss) {
-              next();
-            } else if (routerContext.url) {
-              res.status(routerContext.status || 301);
-              res.set('Location', routerContext.url);
-              routerContext.headers && res.set(routerContext.headers);
-              res.end();
-            } else {
-              res.status(routerContext.status || 200);
-              routerContext.headers && res.set(routerContext.headers);
-              res.type('html');
-              res.send(document);
-            }
-          });
+        const { context = {} } = this.options;
+        if (context.miss) {
```

Comment:
Does this `staticContext` based API (`miss` being true/false, `url` causing a redirect etc.) have any tests and or documentation?

I just wonder how anyone could know about them. The components provided a higher level API that potentially made it easier to use.

Reply:
Absolutely. :)